### DISCUSSION TITLE
Assign users to app using API

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#7377] [Help] [answered]
Hi guys, anyway that I can assign users to access the app through API instead of using manual assign from the management UI
 
![image](https://user-images.githubusercontent.com/597759/185724052-2ad81728-d48c-4b8c-9dd8-bfc4a8831123.png)

**Comment by aderici** [comment, marked as answer]
Yes, see "public api" part of documentation and it shows you examples.
